### ISSUE TITLE
Thumbnails of blogposts are stretched

### ISSUE THREAD

**Issue by monsieurhannes** [repaircafe/malmo.repaircafe.nu#37]
Just noticed that the preview-thumbnails of blogposts are stretched. Any idea how to fix that?
![grafik](https://user-images.githubusercontent.com/20651440/68992696-ea62f500-086e-11ea-9d8b-edc048e9dc5a.png)

**Comment by qerub** [comment]
Good catch. I think the proper solution is to add the CSS rule `object-fit: cover` to the image.
